packages/blade/src/tokens/global/motion.ts

```diff
@@ -0,0 +1,140 @@
+    effective: EasingFunctionFactory | 'cubic-bezier(0.3, 0, 0.2, 1)';
+    /** `cubic-bezier(0.5, 0, 0, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    revealing: EasingFunctionFactory | 'cubic-bezier(0.5, 0, 0, 1)';
+    /** `cubic-bezier(1, 0.5, 0, 0.5)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    wary: EasingFunctionFactory | 'cubic-bezier(1, 0.5, 0, 0.5)';
+  };
+  /** Easings for all entrance animations*/
+  entrance: {
+    /** `cubic-bezier(0.5, 0, 0.3, 1.5)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    attentive: EasingFunctionFactory | 'cubic-bezier(0.5, 0, 0.3, 1.5)';
+    /** `cubic-bezier(0, 0, 0.2, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    effective: EasingFunctionFactory | 'cubic-bezier(0, 0, 0.2, 1)';
+    /** `cubic-bezier(0, 0, 0, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    revealing: EasingFunctionFactory | 'cubic-bezier(0, 0, 0, 1)';
+  };
+  /** Easings for all exit animations*/
+  exit: {
+    /** `cubic-bezier(0.7, 0, 0.5, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    attentive: EasingFunctionFactory | 'cubic-bezier(0.7, 0, 0.5, 1)';
+    /** `cubic-bezier(0.17, 0, 1, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    effective: EasingFunctionFactory | 'cubic-bezier(0.17, 0, 1, 1)';
+    /** `cubic-bezier(0.5, 0, 1, 1)`
+     *
+     *  Returns a `string` of `"cubic-bezier(...)"` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+    revealing: EasingFunctionFactory | 'cubic-bezier(0.5, 0, 1, 1)';
+  };
+};
+
+export type Motion = Readonly<{
+  delay: Delay;
+  duration: Duration;
+  easing: Easing;
+}>;
+
+const delay: Delay = {
+  '2xshort': 70,
+  xshort: 120,
+  short: 180,
+  long: 3000,
+  xlong: 5000,
+};
+
+const duration: Duration = {
+  quick1: 70,
+  quick2: 150,
+  quick3: 200,
+  moderate1: 250,
+  moderate2: 300,
+  gentle1: 400,
+  gentle2: 600,
+  gentle3: 900,
+};
+
+/* makeBezier returns a string of `cubic-bezier()` for web & a react-native-reanimated Easing Function of type `EasingFunctionFactory` for native */
+const easing: Easing = {
```

Comment:
the type and autosuggestion of `Easing` how will that be used in action actually? I see you are directly using `makeBezier` function.

Reply:
Didn't get your point here. But if you're wondering how the typing helps here - if I pass a wrong input to `makeBezier`, it will return an incorrect value that won't fit with the expected easing and throw an error. For example,

<img width="881" alt="Screenshot 2022-04-21 at 10 59 45 AM" src="https://user-images.githubusercontent.com/24487274/164380915-e459bdfd-0380-4b59-88c1-ca5b909369b3.png">